whois-commons/src/main/java/net/ripe/db/whois/common/hazelcast/WhoisHazelcastBlockedIps.java

```diff
@@ -0,0 +1,64 @@
+package net.ripe.db.whois.common.hazelcast;
+
+import com.google.common.base.Joiner;
+import com.hazelcast.collection.ISet;
+import com.hazelcast.core.HazelcastInstance;
+import net.ripe.db.whois.common.ip.IpInterval;
+import net.ripe.db.whois.common.profiles.WhoisProfile;
+import org.eclipse.jetty.util.StringUtil;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+@Component
+@Profile({WhoisProfile.DEPLOYED})
+public class WhoisHazelcastBlockedIps implements HazelcastBlockedIps{
+
+    private static final Logger LOGGER = getLogger(WhoisHazelcastBlockedIps.class);
+
+    private static final Joiner COMMA_JOINER = Joiner.on(',');
+
+    private final ISet<IpInterval> ipBlockedSet;
+
+    public WhoisHazelcastBlockedIps(final HazelcastInstance hazelcastInstance,
+                                    @Value("${ipranges.blocked.list:}") final String blockedListIps) {
+
+        ipBlockedSet = hazelcastInstance.getSet("ipBlockedSet");
+
+        for (final String address : StringUtil.csvSplit(blockedListIps)) {
+            addBlockedListAddress(address);
```

Comment:
Why use csvSplit? How about Stream.of(blockedListIps.split(",")).map(IpInterval::parse).toList() ?

Reply:
We are doing the same in WhoisDoSFilter
I can change this yes...but just want to be consistent

Reply:
This is the suggestion that I was talking before about no using StringUtils.csvSplit
the dependency is no longer necessary so I removed it.
This change is applied as well